### ISSUE TITLE
Usage advices for new users

### ISSUE THREAD

**Issue by johanneszab** [johanneszab/TumblThree#301]
* The correct file to download is the [Application.zip](https://github.com/johanneszab/TumblThree/releases/download/v1.0.8.68/TumblThree-v1.0.8.68-Application.zip).
* Instead of adding one blog after another in the text box at the bottom left corner, use the clipboard monitor! Either:
  * copy (ctrl-c, maybe you'll have to press it twice) the URL from the address bar of your browser, or
  * make a text file with all your blogs / likes / searches and copy (ctrl-c) the whole content (ctrl-a) at once. For example, the blogs to download (example_blogs.txt) in an editor (e.g. Notepad++) would look like this:
~~~~
http://nature-e.tumblr.com/
https://www.tumblr.com/liked/by/wallpaperfx/page/0
mywallpapercollections.tumblr.com
https://www.tumblr.com/tagged/cars?before=1543618889
www.tumblr.com/search/video
https://www.tumblr.com/dashboard/blog/blogname
~~~~
  * Pressing __ctrl-a, ctrl-c__ will add all 6 blogs/tag searches/hidden blogs (the last url in the example) at once.
  * Notice: You cannot simply add the blogs name to the text box in the bottom left corner or the clipboard monitor. You'll need the url (with or without the protocol (i.e. with or without _http(s)://_)) as the blogs name is not sufficient to figure out what you actually want to download (maybe a hidden blog, maybe likes?)

* You need to add a blog to the queue (on the right side) after adding it! To do so, click the "Add to Queue"-button, or double click the blog, or drag and drop the blog over to the queue.

* Do not extract TumblThree to a location that requires admin rights (e.g. C:\Program Files\..). Simply because I've never tested it. If you do, make sure that you change the default download location. Otherwise TumblThree will just not add the blog if it has no permission to write (Thanks to Jason for testing this out).

* [TFA detected ..](https://github.com/johanneszab/TumblThree/issues/307)

* [did you enable to download "image/video/audio meta" for downloading all text posts?](https://github.com/johanneszab/TumblThree/issues/347)

* [the correct format to add a blog](https://github.com/johanneszab/TumblThree/issues/345)

[example_blogs.txt](https://github.com/johanneszab/TumblThree/files/2645664/example_blogs.txt)

**Comment by euloghtos** [comment]
didn't know where to ask, but when is the release which fixes the download of frames instead of videos? Or can i fix it somehow myself?

**Comment by bobblestheninja** [comment]
Does this work on Mac, and if so how exactly would one go about installing it properly to run it

**Comment by c-doves** [comment]
A super simple guide for everyone from the tumblr purge - **To download a tumblr**

1. Download the zip file
2. Unzip it somewhere you can find it
3. Open TumblThree.exe
4. Under settings, choose what you want (video? reblogs? asks?) and also log in for more complete access.
5. Back to the main screen, add tumblrs using their URL (eg. example.tumblr.com). Add them to the queue and click "crawl" to start downloading. It takes a minute to start so be patient!

**Comment by c-doves** [comment]
A super simple guide for writers from the tumblr purge - **To view downloaded text posts**

Text posts (and links, asks, etc) are downloaded as the file type .txt (ie. texts.txt, answers.txt ...) but formatted with a version of html? (I don't code, okay)
I accessed mine by renaming the file. So **texts.txt** got renamed **texts.html**
It helped with most of the text formatting. Some special characters (like quotation marks) didn't convert well, but its mostly in an easy-to-read format. 

To @johanneszab is there a way to add an html output option for the .txt files? Or convert the current files with more complete formatting for things like quotation marks? There is a huge writer community on tumblr scared of losing their posts, but I don't know how much work this takes. Thank you.

**Comment by RevoNYC** [comment]
I have tried this app several times.  The "Add Blog, Remove Blog, Show Files, Remove from Que"
 buttons are all disabled (I can not activate them).  Running Windows 10.  I am logged into Tumblr. Not sure I am running the application correctly or if something else is going on?  Read all the threads, watched videos.. this sounds like it can not be easier but I think there is a bug or a conflict somewhere.  One bit of info, Windows defender pops up and asks if I want to run this app. I have to work around it but the app launches.. buttons do not work.

**Comment by RevoNYC** [comment]
And I am running IE

**Comment by johanneszab** [comment]
>  The "Add Blog, Remove Blog, Show Files, Remove from Que" buttons are all disabled (I can not activate them)

That those buttons are disabled is not an issue. That is the default behavior if there is no blog added to TumblThree. Simply because you cannot remove a blog, show files of a blog or a add/remove from queue a blog if there is no blog added. Thus, buttons do work!

1. Make sure you have write permissions in your Download Location (i.e., notice the admin rights section above).
2. Add a blog to TumblThree. For startes, Do and test what I've written in the first post, and select the urls of the test blogs, copy them to the clipboard, see if they will be added!

**Comment by RevoNYC** [comment]
Ok, adding them worked.  So how do I download my liked pages?

https://www.tumblr.com/liked/by/**"revonyc"**/page/0  for example?

**Comment by DrStephenStark** [comment]
I downloaded and set up the application to try to find and download a few blogs I adore- 
One of the things i found is that one or two of them wouldn't download because of an error saying that the log was offline, is there anything i can do about this, or perhaps a way around it?
I also was wondering if this works for downloading posts with a certain tag Like searching '#Supernatural' or '#Stucky' or something, and perhaps copying something like this? 
https://www.tumblr.com/search/stucky or https://www.tumblr.com/search/voltron

**Comment by johanneszab** [comment]
> One of the things i found is that one or two of them wouldn't download because of an error saying that the log was offline, is there anything i can do about this, or perhaps a way around it?

Maybe a wrong url format? Anything special in the name? A user here send me a 20 000 lines text file full with blogs. Non of those were wrongly classified by TumblThree, nor not crawlable if it actually was online. The last bug of this kind was months ago. Sure, could still be possible, but I'm guessing not. And I also don't have to the url to the blog, so I can only guess and not verify it. 

> I also was wondering if this works for downloading posts with a certain tag Like searching '#Supernatural' or '#Stucky' or something, and perhaps copying something like this?

> I also was wondering if this works for downloading posts with a certain tag Like searching '#Supernatural' or '#Stucky' or something, and perhaps copying something like this?
https://www.tumblr.com/search/stucky or https://www.tumblr.com/search/voltron

Those urls don't work? The results are bad? There also would be the "tag search" as for example https://www.tumblr.com/tagged/supernatural. I've not implemented those searches, Tumblr did. I'm not responsible for crappy results ;)

![search_tumblr](https://user-images.githubusercontent.com/1684561/49682681-d6d4d900-fab8-11e8-86fb-46ce1d00aea6.png)

**Comment by xmiscn** [comment]
I have a "new user" question: I think I set up everything and I find in the download place lots of txt files with links to my photos, but the photos themself are missing. I am sure that images to download is marked, but how do I make the application download them? Is this a 2 phase process?

**Comment by johanneszab** [comment]
@xmiscn:

No, it's not. But you might have selected the checkbox "download url list". In that case, TumblThree will create text files containing the actual urls to the binary (photo, video) files, instead of the actual files themselves. If you have text files ending with *\_url.txt*, then that is the case.
Delete the blogs in question from TumblThree, re-add them, then it will re-download only missing files (i.e. your photos) if you unselect that option.

As a tip, you can do an export of all your currently added blogs to a text file in the Settings. After you've deleted all your blogs you want to re-download from TumblThree, you can simply re-add them by just opening the file and pressing ctrl-a and ctrl-c.

**Comment by BigTime21** [comment]
Ver new user, dumb as a post on this stuff. I got TumblThree downloaded. When i try to open it, File Viewer says I need an upgrade. $30 later I get the upgrade, but it open the .exe file to look like code. What did I do wrong?

**Comment by BigTime21** [comment]
Why did you delete my comment? I was asking for help!

**Comment by johanneszab** [comment]
@BigTime21:

1. How does asking for help fit into "Usage advice"? My thought for this was to keep this issue/thread as short as possible to give an overview about what people often seem to miss. All people just append randomly questions, feature requests and issues to it, then it doesn't fulfill its purpose. There also already is an [thread](https://github.com/johanneszab/TumblThree/issues/112) for this.

2. You question was more a little bit too sketchy/spammy for my taste. TumblThree doesn't cost $30, and I don't know why you/everyone should have payed this. That's why I deleted it. And all the other things I didn't even understand. Just read the advices/readme here on how to use the application, then you should be able to get it going.

**Comment by bearluvr656** [comment]
Hello, Im new to using this and was just wondering if tumblthree helps export my likes as well or is it just the reblogs?

**Comment by BigTime21** [comment]
Forget you. I found a much easier program. Goodbye.

**Comment by pow1234** [comment]
Please help I am a newbie have downloaded the zip file and unzipped but cannot find Open TumblThree.exe or run it in anyway

**Comment by Obergurke** [comment]
I'm sorry, I do keep getting the unparsable error, despite having tried the .68 and then the .63 version.
Funnily enough, the likes get downloaded with no issue.
